Corrected initial commit for Apis melifera

janaobsteter opened the pull request.

grahamgower commented on stdpopsim/catalog/ApiMel/species.py:
```diff
@@ -0,0 +1,117 @@
+    doi="https://dx.doi.org/10.1101%2Fgr.5680406",
+    year=2006,
+    author="Beye et al.",
+    reasons=stdpopsim.CiteReason.REC_RATE,
```
Hi @janaobsteter. The reason the tests are failing is because the "reasons" fields for the citations needs to be a `set` of CiteReasons. So above, this needs to be `reasons={stdpopsim.CiteReason.REC_RATE}`, with curly braces. Similarly for the other `reasons=` lines.

janaobsteter replied:
Cool, thanks! Will correct this!

jeromekelleher commented on stdpopsim/catalog/ApiMel/demographic_models.py:
```diff
@@ -0,0 +1,26 @@
+# import msprime
```
We should probably leave this file out of the initial species addition I think - we can follow up with other PRs for demographic models.

janaobsteter replied:
Yes, agreed, we are still working on the models anyway

janaobsteter replied:
@jeromekelleher , it took me some time to come back to this but I have corrected some mistakes in the species.py file, added some references and also removed the demographic model from the git. Could you please revise this version? Thanks!

jeromekelleher commented on stdpopsim/catalog/ApiMel/species.py:
```diff
@@ -0,0 +1,117 @@
+
+from . import genome_data
+
+# [The following are notes for implementers and should be deleted
```
We can delete this comment (and others like it), it's just there as an initial guide for people.

jeromekelleher reviewed (commented):
Looks great, thanks! Some minor comments.

jeromekelleher commented:
The updates look great @janaobsteter, thanks! However, it would be good to get the basic species PR merged first without the demographic model. If you're unsure of the git process for this, I'm happy to talk you through it.

janaobsteter commented:
@Jerome Kelleher ***@***.***> , thanks! I will start the
process and write on Slack when I run into issues.


On Fri, 9 Apr 2021 at 16:46, Jerome Kelleher ***@***.***>
wrote:

> The updates look great @janaobsteter <https://github.com/janaobsteter>,
> thanks! However, it would be good to get the basic species PR merged first
> without the demographic model. If you're unsure of the git process for
> this, I'm happy to talk you through it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/popsim-consortium/stdpopsim/pull/815#issuecomment-816735859>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AELB25ZPHL3JYLKJNGEZ73TTH4HNTANCNFSM4ZVC4BAQ>
> .
>

janaobsteter commented:
@jeromekelleher , it took me some time to come back to this but I have corrected some mistakes in the species.py file, added some references and also removed the demographic model from the git. Could you please revise this version? Thanks!
